packages/rolldown/build.ts

```diff
@@ -105,8 +105,6 @@ function withShared(
       /rolldown-binding\..*\.wasm/,
       /@rolldown\/binding-.*/,
       /\.\/rolldown-binding\.wasi\.cjs/,
-      // some dependencies, e.g. zod, cannot be inlined because their types
-      // are used in public APIs
```

Comment:
I saw this comment, but I couldnt find any types that use valibot, so I asssume it no longer applies

Reply:
We used to drived types from zod-schema, so we need the zod's types. Now we declare types manually, so it won't be an issue now.